test/Microsoft.Extensions.Options.Test/OptionsTest.cs

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Its a rename/move, I stash/unstashed since I made the initial changes in the wrong branch which might have confused git